docs/source/usage_guides/quantization.md

```diff
@@ -28,7 +28,7 @@ You will need to install the following requirements:
 
 - Install `bitsandbytes` library
 ```bash
-pip install bitsandbytes==0.39.0
+pip install bitsandbytes
```

Comment:
Considering we're facing issues with versions, are we marking that down in the source of accelerate for checking minimum `bnb` versions when something like this happens?

Reply:
We do have check for minimum version of bnb when the user tries to do 4-bit quant or 8-bit quant:

Reply:
These need to be updated for 0.40.0 for the latest breakage though, right? (Unrelated to this exact PR, just something I'm wondering about with the fixes. Otherwise PR works fine for me :) )

We also might want to consider `pip install accelerate[bnb]` at some point potentially since it is technically an extra

Reply:
Excellent 🥳